api/src/main/java/uk/gov/hmcts/PaymentApiApplication.java

```diff
@@ -58,7 +68,10 @@ public CacheManager cacheManager() {
                     .build()),
             new CaffeineCache("sites", Caffeine.newBuilder()
                 .expireAfterWrite(48, TimeUnit.HOURS)
-                .build())
+                .build()),
```

Comment:
@amitchidrewar1301 @AnilDhawaleGitHub -  I think this was added as part of OIDC changes. I don't think so we need this change for spring boot upgrade.

Reply:
removed